src/deploy/project.py

```diff
@@ -279,6 +275,9 @@ def _ecs_deploy(service, deployment_label):
         )
 
         for image_id, image_name in sorted(release['images'].items()):
+            if image_name is None:
+                continue
```

Comment:
Is it worth logging some sort of warning here?

Reply:
I don't think we'd log a warning there - by convention the click display logic is in `deploy.py`. I'm not even sure it's something to warn about as the case where only some services have changed will be fairly common.

However it is not particularly clear when we display a release (https://github.com/wellcomecollection/weco-deploy/pull/113/files#diff-48252ce381e5fcfcff5ddf9bc65a66cd33f6c84e8d4028525727ec1f212d3741R325) that there is no image for a service.

I'll take a look at having that view display what's going on.